src/containers/ReportArticle/index.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import Materialize from "materialize-css";
+import PropTypes from 'prop-types';
+import swal from 'sweetalert';
+import Modal from "../../components/Modal";
+import DropDown from "../../components/DropDown";
+import DropDownItem from "../../components/DropDownItem";
+import { REPORT_WITHOUT_DATA } from './state/types';
+
+
+export class ReportArticle extends Component {
+  constructor(props) {
+    super(props);
+    this.serverMessage = this.serverMessage.bind(this);
+  }
+
+  componentDidMount = () => {
+    Materialize.Modal.init(document.querySelector('#report-modal'), {});
+  };
+
+  componentDidUpdate = () => {
+    Materialize.FormSelect.init(document.querySelectorAll('select'), {});
+  };
+
+
+  state = {
+    type: "",
+    description: "",
+  };
+
+  serverMessage(history) {
+    const { articleReporting } = this.props;
+    const {
+      success, hasErrors, errors, message,
+    } = articleReporting;
+
+    if (success) {
+      swal("Article Reporting", message, "success");
+      history.push("/");
+    } else if (hasErrors) {
+      if (errors === REPORT_WITHOUT_DATA) {
```

Comment:
Add test for the `success === false` scenario

Reply:
There is a test for this:
```
it('should toast when no data provided', () => {
    wrapper.setProps({ hasErrors: true, errors: REPORT_WITHOUT_DATA, success: false });
    instance.serverMessage(historyMock);
    expect(historyMock.push).toHaveBeenCalledTimes(1);
  });
```